doc/Programming-guide/Key-platform-features/Common-Type-Library/index.md

```diff
@@ -71,80 +76,124 @@ Matching FQN in a different Tenant | N/A | OK | OK
+Use the **Display higher scopes** checkbox to toggle visibility of the System CTs.
+
+As [Tenant developer]({{root_url}}Glossary/#tenant-developer), you can get the list of available Tenant CTs and System CTs in the same way as Tenant administrator.
+In addition, Tenant developer can get the list of available application CTs using the [REST API]({{root_url}}Programming-guide/Server-REST-APIs/#!/Common_Type_Library/getApplicationLevelCTLSchemasByAppToken) call or by clicking **Application CTL** on the **Administration UI** page.
+Use the **Display higher scopes** checkbox to toggle visibility of the system and tenant CTs.
+
+### Create a new CT
+
+To create a new CT:
+
+- Use the [REST API]({{root_url}}Programming-guide/Server-REST-APIs/#!/Common_Type_Library/saveCTLSchemaWithAppToken) call.
 
-New CT can be created via [REST API]({{root_url}}Programming-guide/Server-REST-APIs/#!/Common_Type_Library/saveCTLSchemaWithAppToken) call or by clicking **Add new type** in corresponding CTL window from Administration UI.
+OR
 
-> **NOTE:** The Kaa admin can add new System CTs.
-The Tenant admin can add new Tenant CTs.
-The Tenant developer can add new Application CTs. 
+- Open the **Administration UI** page, select the corresponding CTL and click the **Add new type** button.
+
+>**NOTE:** Kaa administrator creates new system CTs.
+>Tenant administrator creates new tenant CTs.
+>Tenant developer creates new application CTs.
+{:.note}
+
+In the **Add new type** window, fill in all the required fields and click **Add**.
 
-In **Add new type** window fill in all the required fields and click **Add** to apply the changes.
-Below is shown an example of **Add new type** window for Tenant CT.
 ![Create Tenant CTL](attach/create_tenant_ctl.png)
-There is also ability to create a new version of CT by clicking **Create new version** at the top of **CT details** window which is described in the next chapter.
 
-## Get or view CT details
+### View CT details
+
+To view the CT details:
+
+- Use the [REST API]({{root_url}}Programming-guide/Server-REST-APIs/#!/Common_Type_Library/getCTLSchemaById) call.
+
+OR
+
+- Open the **Administration UI** page, select the corresponding CTL and click on the CT in the list.
+The **Common type details** page will open.
 
-CT details can be obtained via [REST API]({{root_url}}Programming-guide/Server-REST-APIs/#!/Common_Type_Library/getCTLSchemaById) call or can be viewed in **CT details** window of Administration UI.
-To open CT details click the row of the CTs list in corresponding CTL window.
-Below is shown an example of **CT details** window for Application CT.
 ![View Application CTL](attach/view_application_ctl.png)
-To view another version of the CT, click on the version drop-down and select the appropriate version.
 
-## Delete CT schema
+To view another version of the CT, select it form the **Version** drop-down list.
+To create a new version of the CT, click **Create new version**.
+
+### Delete CT schemas
```

Comment:
Actually, you only delete versions, not the whole CTs. You can delete the whole CT when you delete the last available schema in it. But when you click the 'Delete' button on the 'Common type details' page, you only delete a schema version (and the subsequent warning message says the same).